app/modules/kit-library/assets/js/hooks/use-cloud-kits-eligibility.js

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from 'react-query';
+
+export const KEY = 'cloud-kits-availability';
+
+/**
+ * Fetch cloud kits availability from WordPress backend
+ *
+ * @return {Promise<boolean>} Whether cloud kits should be available
+ */
+async function fetchCloudKitsEligibility() {
+	const response = await $e.data.get( 'kits-cloud-eligibility/index', {}, { refresh: true } );
+
+	const isCloudKitsExperimentActive = elementorCommon?.config?.experimentalFeatures?.e_cloud_library_kits;
```

Comment:
Early return to reduce number of calls if experiment is not active?
```suggestion
	const isCloudKitsExperimentActive = elementorCommon?.config?.experimentalFeatures?.e_cloud_library_kits;
	 if( ! isCloudKitsExperimentActive ) {
	 return false;
	 }
	
	const response = await $e.data.get( 'kits-cloud-eligibility/index', {}, { refresh: true } );
```

Reply:
YEs, good catch!